code/modules/modular_computers/file_system/programs/ntdownloader.dm

```diff
@@ -30,8 +37,8 @@
 	if(!PRG || !istype(PRG))
 		return 0
 
-	// Attempting to download antag only program, but without having emagged computer. No.
-	if(PRG.available_on_syndinet && !(computer.obj_flags & EMAGGED))
+	// Attempting to download antag only program, but without having emagged/syndicate computer. No.
+	if(PRG.available_on_syndinet && !((computer.obj_flags & EMAGGED) || (computer.device_theme = "syndicate")))
```

Comment:
Could you make the syndicate computer property dissociated from the visual theme, and instead its own thing?

Reply:
Rewrote this section to use a variable in the nt downloader rather than whether the system is emagged or uses syndicate style. Syndicate versions simply enable the emagged var by default.